src/annotator/config/settings.js

```diff
@@ -78,23 +83,24 @@ export function settingsFrom(window_) {
       return null;
     }
 
-    return jsonConfigs.group || groupFromURL();
+    return checkIfString(jsonConfigs.group) || groupFromURL();
   }
 
-  // TODO: Move this to a coerce method
   function showHighlights() {
```

Comment:
Previously this function could have returned arbitrary values, if the value in the configuration was not one of the allowed cases. Now it always returns either `always`, `never` or `whenSidebarOpen`.